frontend/src/components/Bar/bar.scss

```diff
@@ -0,0 +1,85 @@
+@import "../../vars";
+
+.navigation {
+  position: fixed;
+  left: 0;
+  top: 0;
+  z-index: 2;
+  width: 100vw;
+  overflow: hidden;
+  font-size: 0.9em;
+  background-color: $background_second_color;
+  border-top: 2px solid $company_color;
+
+  > .items {
+    padding: 0;
+    width: fit-content;
+    list-style: none;
+    margin: 0 auto;
+
+    > .logo {
+      position: fixed;
+      top: 0;
+      left: 0;
+      width: 100vw;
+      background: $background_second_color;
+      padding: 1em 0 1em 1em;
+      border-bottom: 2px solid $company_color;
+
+      > img {
+        display: inline-block;
+        height: 3.5em;
+      }
+    }
+
+    > .link {
+      color: inherit;
+      text-decoration: inherit;
+
+      > .item {
+        display: block;
+        transition: all 0.3s ease-in-out;
+        padding: 1.5em 1.25em 1.5em 1.25em;
+        text-align: center;
+        float: left;
+
+        > .icon {
+          /* Zeit für einfahren */
+          transition: all 0.4s ease-in-out;
+          filter: invert(87%) sepia(0%) saturate(98%) hue-rotate(137deg)
+            brightness(89%) contrast(82%);
+          width: 1.5em;
+        }
+
+        > .title {
+          display: none;
+        }
+      }
+
+      &.active .item {
+        background-color: $company_color;
+        color: $font_light_color;
+        box-shadow: 0 14px 28px rgba(0, 0, 0, 0.25),
+          0 10px 10px rgba(0, 0, 0, 0.22);
+
+        > .icon {
+          filter: invert(100%) sepia(0%) saturate(7499%) hue-rotate(140deg)
+            brightness(104%) contrast(101%);
+        }
+      }
+
+      &:not(.active) {
+        > .item:hover {
+          > .icon,
+          > .title {
+            @media only screen and (min-width: $breakpoint) {
+              /* Zeit für ausfahren */
+              transition: all 0.2s ease-in-out;
```

Comment:
Kann man das in einem Theme Provider oder so machen? Ich kann mich nicht mehr ganz sicher wie genau das war, aber dann hätten wir einheitliche Animationen und so